OpenSim/Tools/AnalyzeTool.cpp

```diff
@@ -401,7 +401,15 @@ loadStatesFromFile(SimTK::State& s)
         _model->getSimbodyEngine().convertDegreesToRadians(*qStore);
         _model->getSimbodyEngine().convertDegreesToRadians(*uStore);
 
-        _statesStore = createStatesStorageFromCoordinatesAndSpeeds(*_model, *qStore, *uStore);
+        // used to use createStatesStorageFromCoordinatesAndSpeeds(*_model, *qStore, *uStore);
+        double ti = qStore->getFirstTime();
+        double tf = qStore->getLastTime();
+        uStore->addToRdStorage(*qStore, ti, tf);
+
+        delete _statesStore;
+        _statesStore = new Storage(512, "states");
+
+        _model->formStateStorage(*qStore, *_statesStore);
```

Comment:
I changed the implementation of `loadStatesFromFile` to use an existing method ( `Model::formStateStorage`) internally with no change to the interface. If the GUI relies on side-effects rather than an interface then their could be issues. I hope this would improve consistency between `AnalyeTool` and other Tools that use `Model::formStateStorage`.